README.md

```diff
@@ -10,7 +10,7 @@ In order to get started:
 $ git clone git@github.com:yarnpkg/website.git yarn-website
 $ cd yarn-website
 ```
-
+Prerequisites
```

Comment:
could you make this a `##`, and move the "and then" part into a heading called "developing"?